### PR TITLE
Updated the doc with new the options from onboard_agent script

### PR DESCRIPTION
**Pull request by arjun-urs**
Purge and Proxy options updated in the doc

@keikhara @NarineM

### CODE REVIEW

File: docs/OMS-Agent-for-Linux.md [hunk 1]
```diff
@@ -686,7 +690,7 @@ The default settings are
 ```
 
 # Uninstalling the OMS Agent for Linux
-The agent packages can be uninstalled using dpkg or rpm, or by running the bundle .sh file with the `--remove` argument. Additionally, if you want to completely remove all pieces of the OMS Agent for Linux you can run the bundle .sh file with the `--purge` arguement. 
+The agent packages can be uninstalled using dpkg or rpm, or by running the bundle .sh file with the `--remove` argument. Additionally, if you want to completely remove all pieces of the OMS Agent for Linux you can run the bundle .sh file (or onboard_agent.sh file) with the `--purge` argument.
```
Comment (NarineM) [inline docs/OMS-Agent-for-Linux.md]: hi, can you change the "remove all pieces" to be "to remove user data such as certificate and agent configurations"?

Comment (arjun-urs) [reply docs/OMS-Agent-for-Linux.md]: Done

File: docs/OMS-Agent-for-Linux.md [hunk 2]
```diff
@@ -686,7 +690,7 @@ The default settings are
 ```
 
 # Uninstalling the OMS Agent for Linux
-The agent packages can be uninstalled using dpkg or rpm, or by running the bundle .sh file with the `--remove` argument. Additionally, if you want to completely remove all pieces of the OMS Agent for Linux you can run the bundle .sh file with the `--purge` arguement. 
+The agent packages can be uninstalled using dpkg or rpm, or by running the bundle .sh file with the `--remove` argument. Additionally, if you want to completely remove user data such as certificate and agent configurations of the OMS Agent for Linux you can run the bundle .sh file (or onboard_agent.sh file) with the `--purge` argument.
```
Comment (NarineM) [inline docs/OMS-Agent-for-Linux.md]: Thanks, one more thing, can you rephrase "if you want to completely remove user data such as certificate and agent configurations of the OMS Agent for Linux you can run the bundle .sh file "

to be "if you want to remove user data such as certificate and agent configurations of the OMS Agent for Linux you need to run the bundle .sh file "

Comment (arjun-urs) [reply docs/OMS-Agent-for-Linux.md]: Done
